lizard.py

```diff
@@ -688,6 +688,13 @@ def print_warnings(option, scheme, warnings):
     return warning_count, warning_nloc
 
 
+def print_no_warnings(option):
+    warn_str = "No thresholds exceeded ({0})".format(
+        ' or '.join("{0} > {1}".format(
+            k, val) for k, val in option.thresholds.items()))
```

Comment:
This is the exact same formatting as `print_warnings()` on line 679 right above here so I kept it for consistency in hopes that the PR will be more easily adopted.